api_tests/providers/preprints/views/test_preprint_provider_moderator_list.py

```diff
@@ -129,7 +129,7 @@ def test_list_post_admin_success_email(self, mock_send_grid, app, url, nonmodera
 
     def test_list_moderators_alphabetically(self, app, url, admin, moderator, provider):
         admin.fullname = 'Alice Alisdottir'
-        moderator.fullname = 'Bob Bobsson'
+        moderator.fullname = 'Bob Bobson'
```

Comment:
This spelling was more common and there didn't seem to be some intent to use two different spellings (unlike Freddy)